src/app/session.service.ts

```diff
@@ -1,15 +1,35 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { Http } from '@angular/http';
+import { GlobalSettingsService } from './globalSettings.service';
 
 @Injectable()
 export class SessionService {
+  config;
+  constructor(private http: Http, private globalSettingsService: GlobalSettingsService ) {
+    this.config = globalSettingsService.get();
+  }
+
+  create(): Observable<any> {
```

Comment:
I would be happy for this to be in another PR. I think it will be more of a concern once we start testing the system.

Should we raise an issue or just put in another PR?